app/components/file_attachment_list/file_attachment_image.js

```diff
@@ -78,6 +79,8 @@ export default class FileAttachmentImage extends PureComponent {
         } else if (file.id) {
             imageProps.thumbnailUri = Client4.getFileThumbnailUrl(file.id);
             imageProps.imageUri = Client4.getFilePreviewUrl(file.id);
+            imageProps.miniPreview = file.mini_preview;
```

Comment:
if the file.mini_preview is available, you could use it instead of the `thumbnailUri`. Basically set the thumbnail URI to the base64 string as you should also have the mimeType in the **file** object.

Then make the appropriate changes in `ProgressiveImage`

Reply:
I didn't quite get this, can you elaborate please? basically my doubt is, if I set base 64 in thumbnailUri, how will the progressiveImage component know that it has to load original image on entering the screen?

Reply:
@bhargav50, are you ok if I push some changes to you PR ?

Reply:
Yeah sure